src/Todoist.Net/Serialization/Resolvers/JsonResolverModifiers.cs

```diff
@@ -53,17 +45,72 @@ public static void FilterSerializationByType(JsonTypeInfo typeInfo)
                 {
                     propertyInfo.ShouldSerialize = (obj, value) => false;
                 }
+            }
+        }
+
+        public static void IncludeUnsetProperties(JsonTypeInfo typeInfo)
```

Comment:
We should probably write a test for the new serialization logic.

Reply:
Done.